scenarios/scenario-10/README.md

```diff
@@ -0,0 +1,116 @@
+     - _validateQueryPrams.bal_ file consists with _validateRequest_ function which modify the request flow. we have added the function name _validateRequest_ in the swagger definition as follow.
+     ```
+         x-wso2-request-interceptor: validateRequest
+        
+     ```
+     - _modifyResponse.bal_ file consists with _validateResponse_ function which modify the response flow. we have added the function name _validateResponse_ in the swagger definition as follow.
+     ```
+         x-wso2-response-interceptor: validateResponse
+     ```
+     
+- Execute the following to expose pet-store as an API.
+
+- Create API <br /> 
+    ```
+        apictl add api -n petstore-int --from-file=./product-apim-tooling/import-export-cli/build/target/apictl/petstore-int --replicas=1
+    ``` 
+    - Output:
+    ```
+        creating configmap with swagger definition
+        configmap/petstore-int-swagger created
+        creating configmap with interceptors
+        configmap/petstore-int-interceptors created
+        api.wso2.com/petstore-int created
+    ```
+  
+
+- Get service details to invoke the API. (Please wait until the external-IP is populated in the corresponding service)
+    ```
+        apictl get services
+    ```
+    - Output:
+    
+    ```
+        NAME            TYPE           CLUSTER-IP   EXTERNAL-IP       PORT(S)                         AGE
+        petstore-int   LoadBalancer   10.83.4.44   104.197.114.248   9095:30680/TCP,9090:30540/TCP   8m20s
+    ```
+    - You can see petstore-int service has been exposed as a managed API.
+    - Get the external IP of the managed API's service
+ 
+- Invoking the API <br />
+    ```
+        TOKEN=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6IlpqUm1ZVE13TlRKak9XVTVNbUl6TWpnek5ESTNZMkl5TW1JeVkyRXpNamRoWmpWaU1qYzBaZz09In0=.eyJhdWQiOiJodHRwOlwvXC9vcmcud3NvMi5hcGltZ3RcL2dhdGV3YXkiLCJzdWIiOiJhZG1pbkBjYXJib24uc3VwZXIiLCJhcHBsaWNhdGlvbiI6eyJvd25lciI6ImFkbWluIiwidGllciI6IlVubGltaXRlZCIsIm5hbWUiOiJzYW1wbGUtY3JkLWFwcGxpY2F0aW9uIiwiaWQiOjV9LCJzY29wZSI6ImFtX2FwcGxpY2F0aW9uX3Njb3BlIGRlZmF1bHQiLCJpc3MiOiJodHRwczpcL1wvd3NvMmFwaW06OTQ0M1wvb2F1dGgyXC90b2tlbiIsInRpZXJJbmZvIjp7fSwia2V5dHlwZSI6IlBST0RVQ1RJT04iLCJzdWJzY3JpYmVkQVBJcyI6W10sImNvbnN1bWVyS2V5IjoiOFpWV1lQYkk2Rm1lY0ZoeXdVaDVVSXJaNEFvYSIsImV4cCI6MzcxODI5OTU1MiwiaWF0IjoxNTcwODE1OTA1LCJqdGkiOiJkMGI2NTgwNC05NDk3LTQ5ZjktOTcxNC01OTJmODFiNzJhYjMifQ==.HYCPxCbNcALcd0svu47EqFoxnnBAkVJSnCPnW6jJ1lZQTzSAiuiPcGzTnyP1JHodQknhYsSrvdZDIzWzU_mRH2i3-lMVdm0t43r-0Ti0EdBSX2756ilo266MVeWhxbz9p3hPm5ndDCoo_bfB4KbjigjmhXv_PJyUMuWtMo669sHQNs5FkiOT2X0gzFP1iJUFf-H9y762TEIYpylKedVDzQP8x4LCRZsO54e1iA-DZ5h5MKQhJsbKZZ_MMXGmtdo8refPyTCc7HIuevUXIWAaSNRFYj_HZTSRYhFEUtDWn_tJiySn2umRuP3XqxPmQal0SxD7JiV8DQxxyylsGw9k6g==
+    ```
+   
+    ```
+        curl -X GET "https://10.83.4.44:9095/petstore/v1/pet/findByStatus?status=available"  -H "accept: application/xml" -H "Authorization:Bearer $TOKEN" -k
```

Comment:
10.93.4.44 is cluster IP. this should be external IP.

Reply:
changed